recipe/build.sh

```diff
@@ -1,19 +1,37 @@
+#!/usr/bin/env bash
+set -eux
+
+# Add hunspell dictionary paths relative to PREFIX. We have to do this here so
+# that $PREFIX gets expanded into the patch.
+patch -p1 <<EOF
+diff --git a/src/tools/hunspell.cxx b/src/tools/hunspell.cxx
+--- a/src/tools/hunspell.cxx
++++ b/src/tools/hunspell.cxx
+@@ -116,6 +116,8 @@
+ #include "../parsers/odfparser.hxx"
+
+ #define LIBDIR                \
++  "${PREFIX}/share/hunspell:" \
++  "${PREFIX}/share/hunspell_dictionaries:" \
+   "/usr/share/hunspell:"      \
+   "/usr/share/myspell:"       \
+   "/usr/share/myspell/dicts:" \
+
+EOF
+
 autoreconf -vfi
-./configure --prefix=$PREFIX --with-readline --with-ui
-make
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
-make check
+
+./configure \
+    --prefix="${PREFIX}" \
+    --with-readline \
+    --with-ui
+
+make "-j${CPU_COUNT}"
+
+if [[ "${target_platform}" == "${build_platform}" ]]; then
+    make check
 fi
+
 make install
-mv $PREFIX/bin/hunspell $PREFIX/bin/.hunspell
-
-# We have to make a wrapper to add a spelling dictionary path to hunspell,
-# since none of the default ones are relative to the binary (i.e., can be
-# installed as a conda package)
-cat <<EOF > $PREFIX/bin/hunspell
-#!/bin/sh
-export DICPATH='$PREFIX/share/hunspell_dictionaries'
-$PREFIX/bin/.hunspell "\$@"
-EOF
 
-chmod a+x $PREFIX/bin/hunspell
+chmod a+x "${PREFIX}/bin/hunspell"
```

Comment:
Make will install hunspell with executable permissions so this can be removed

Reply:
Sure, reasonable, and would be caught in test if we're wrong!

Reply:
[56c0380](https://github.com/conda-forge/hunspell-feedstock/pull/13/commits/56c03802d918a57de322a3d9a9f43836ed808c0f)